pkg/mqtrigger/messageQueue/kafka.go

```diff
@@ -227,7 +225,7 @@ func kafkaMsgHandler(kafka *Kafka, producer sarama.SyncProducer, trigger *fv1.Me
 		kafka.logger.Error("failed to create HTTP request to invoke function",
 			zap.Error(err),
 			zap.String("function_url", url))
-		return false
+
```

Comment:
Hey @viveksinghggits , you cannot just remove the return here and other places directly, they still need to return but without the bool...

Reply:
Please check the code and its purpose again, thanks.

Reply:
My Bad @life1347, I understand and will make the changes.